apps/web-portal-client/lib/context/ApiProvider.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import {createClient} from '@util-web-api-client'
+import {ApiContext} from './ApiContext'
+
+export function ApiProvider(props) {
+  const client = createClient()
```

Comment:
let's move 'createClient' to 'index.jsx', so we make sure that API created on 100% before we start rendering, 
also, it would simplify the configuration of ApiClient, because other wise we need to pass the same configs throug the ApiContext component's props

Reply:
done